prime-router/docs/onboarding-users/receivers.md

```diff
@@ -1,12 +1,307 @@
 # How to Onboard a New Organization to Receive Data
 Add subsections that mimic the linked resources: ([Brandon’s version](https://docs.google.com/document/d/1noB3lK2Nc_vbD4s5ZHgdTjgIjhCii63x_2bjBz7GM1I/edit#heading=h.be9yxi8thtdw), [Github version](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/how-to-onboard-a-sender.md))
 
-## Receiver Configuration
-
-- Transport Protocols
-  - SFTP
-    - SSH Keys
-  - Azure Blob
-  - REST
-  - SOAP
-  - …
+## Welcome
+
+Our goal is to onboard as many states and local jurisdictions as we can, to receive Hub data!
+This is our internal documentation for how we _currently_ do that onboarding work.
+
+
+The main goal of receiver onboarding is to establish a connection to send data from ReportStream to the STLT.  
+In order to do so there are multiple receiver configurations that need to be configured.  
+During pre-onboarding ideally we will be able to know/collect/obtain all the unknown variables in order to set up the receiver configurations.
+
+## Pre-Onboarding
+* Identify how they want to receive data
+* Determine if we need to be whitelisted prior to connecting
+* Identify STLT Specific (HL7) values (e.g. MSH-5, MSH-6, etc)
+* Determine if they need/want specific data quality or condition filters
+* Determine how they want to receive AoE questions
+* Create Okta Accounts (probably once they are set up fully).
+
+## Table of Contents
+1.    Set up new organization
+2.    Set up new schema
+3.    Generate Test Data
+4.    Test and commit, and deploy to Test and maybe Prod
+5.    Testing in your Docker container
+6.    Create access to the Download site
+7.    Validation in Prod
+8.    Set up transport SSH key for receiver using SFTP
+
+### 1. Set up new organization
+* Create a new branch in git for your changes.
+* Create a new organization for the State, (canonical style: `lt-phd`), in organizations.yml, which is used by your local 
+commandline ./prime cli tool.
+* Follow the pattern of another existing organization.  Carefully set the initial 
+jurisdiction-filter so that data is limited to that state.  (The jurisdiction: STATE and  stateCode: XY  fields should 
+soon provide better enforcement of this)
+* The new organization must have at least one `receiver` defined, but the `receiver` does not need to have a transport 
+defined - the download site can be used until an automated delivery mechanism is set up.
+* There are two fields that will be used as "keys" to further work, below.   The `-name` of the organization 
+(eg, `lt-phd`) will be used as the Okta `group`, and the `translation.schemaName:` value (eg, `lt/lt-covid-19`) will be 
+used as the schema name in the next step.
+* Below is an example of the organization file
+  
+```yaml
+- name: lt-pdh
+  description: LT Department of Health
+  jurisdiction: STATE
+  stateCode: LT
+  receivers:
+    - name: elr
+      topic: covid-19
+      jurisdictionalFilter: [ "matches(ordering_facility_state, LT)" ]
```

Comment:
```suggestion
      jurisdictionalFilter: [ "(%performerState.exists() and %performerState = 'LT') or (%patientState.exists() and %patientState = 'LT')" ]
```

Reply:
In the UP we use FHIR path for filters. %performerState and %patientState are shorthand FHIR paths defined in `metadata/tables/local/fhirpath_filter_shorthand.csv`